api/pyproject.toml

```diff
@@ -39,12 +39,12 @@ pytest = "^8.3.2"
 hypercorn = {extras = ["trio"], version = "^0.17.3"}
 
 pyjwt = "^2.8.0"
-passlib = {extras = ["bcrypt"], version = "^1.7.4"}
 python-jose = {extras = ["cryptography"], version = "^3.3.0"}
 aiofiles = "^23.2.1"
 python-multipart = "^0.0.9"
 debugpy = "^1.8.1"
 sqlalchemy = "^2.0.30"
+bcrypt = "^4.2.0"
```

Comment:
We should use the build-in hashlib in Python, it has `PBKDF2`.

Reply:
In the original impl, passlib was abstracting bcrypt, so all passwords are currently bcrypt hashed. Changing the underlying encryption will break existing user accounts, and there's no password reset functionality (yet) to recover those accounts.

Also depending on who you ask, bcrypt>pbkdf2 for its resistance to GPU brute-forcing... but between the three, scrypt is the best for modern systems.
Still, no ability to reset passwords, so I'm leery of changing the hashing algo and even bcrypt is overkill for a project of this scope

Reply:
There hasnt been any releases, so it won't break anyones system.

Reply:
@Jonpro03 The build-n hashlib has scrypt, that works for me. 

https://docs.python.org/3/library/hashlib.html#hashlib.scrypt